src/Analyzers/Internal.AspNetCore.Analyzers/test/PubternabilityAnalyzerTests.cs

```diff
@@ -34,7 +34,6 @@ public PubternabilityAnalyzerTests(ITestOutputHelper testOutputHelper) : base(te
 
         [Theory]
         [MemberData(nameof(PublicMemberDefinitions))]
-        [QuarantinedTest]
```

Comment:
I'm unable to determine the owner for this test.

Reply:
Leave it quarantined but add the linked issue? https://github.com/dotnet/aspnetcore/issues/22440